web/themes/custom/pineapple/sass/pineapple.styles.scss

```diff
@@ -8,3 +8,4 @@
 @import "base/__base.scss";
 @import "layouts/__layouts.scss";
 @import "components/__components.scss";
+@import "content/__content.scss";
```

Comment:
There's no need to create a new directory, use what's already there. Remove this.

Reply:
We have already discussed.

Reply:
Ermm no we haven't, why is this here?

Reply:
It's okay, we all make mistakes